src/controller/python/chip/ChipBleBase.py

```diff
@@ -25,10 +25,9 @@
 import abc
 import optparse
 import shlex
-import six
 
 
-class ChipBleBase(six.with_metaclass(abc.ABCMeta, object)):
+class ChipBleBase(metaclass=abc.ABCMeta):
```

Comment:
Maybe add `#!/usr/bin/env python3` to start of this file?  And similar for other files in this PR.

Reply:
existing files have no `#!` entry at all, so nothing to change. I would prefer not to add them if they do not existing, to not indicate that they are somehow stand alone executable scripts.